### PR TITLE
Adding numerous adopters

### PR DESCRIPTION
**Pull request by tylerhannan**
Adding MOAR

### CODE REVIEW

File: docs/en/about-us/adopters.md [hunk 1]
```diff
@@ -63,6 +65,7 @@ The following list of companies using ClickHouse and their success stories is as
 | [Citadel Securities](https://www.citadelsecurities.com/) | Finance | — | — | — | [Contribution, March 2019](https://github.com/ClickHouse/ClickHouse/pull/4774) |
 | [Citymobil](https://city-mobil.ru) | Taxi | Analytics | — | — | [Blog Post in Russian, March 2020](https://habr.com/en/company/citymobil/blog/490660/) |
 | [ClickVisual](https://clickvisual.gocn.vip/) | Software | Logging Platform | — | — | [Blog Post, May 2022](https://golangexample.com/a-light-weight-log-visual-analytic-platform-for-clickhouse/) |
+| [Clog](https://www.sendinblue.com/) | Software & Technology | Logging | — | — | [Blog, February 2023](https://www.hybridlogic.co.uk/2023/02/clog/) |
```
Comment (DanRoscigno) [inline docs/en/about-us/adopters.md:68]:
```suggestion
| [Clog](https://www.hybridlogic.co.uk/) | Software & Technology | Logging | — | — | [Blog, February 2023](https://www.hybridlogic.co.uk/2023/02/clog/) |
```

**Review by DanRoscigno — APPROVED** [review]

**Comment by DanRoscigno** [comment]
Thanks so much, Tyler!

File: docs/en/about-us/adopters.md (hunk 1 above)
Comment (tylerhannan) [reply docs/en/about-us/adopters.md]: ooof. sorry. thank you.

Comment (DanRoscigno) [reply docs/en/about-us/adopters.md]: > ooof. sorry. thank you.

I have done that more times than I have fingers

**Pull request merged**